tests/certification/bindings/azure/cosmosdb/README.md

```diff
@@ -0,0 +1,21 @@
+# Azure CosmosDB Binding certification testing
+
+This project aims to test the Azure CosmosDB binding component under various conditions.
+
+## Test plan
+
+### Authentication tests
+
+* Authenticate with Azure Active Directory using Service Principal Client Secret
+* Authenticate with Master Key
+
+### Other tests
+
+* Client reconnects (if applicable) upon network interruption
```

Comment:
This is pending yes? Or does it work by nature of the two included tests passing?

Reply:
It's TODO. Planning to update this markdown more.

Reply:
I'd prefer if we marked it as TODO here then. I see this file as a quick stop to know what the certification tests are actively/planning on testing.